### DISCUSSION TITLE
Does <model-viewer> work on Safari desktop and mobile?

### DISCUSSION THREAD

**Discussion by elliottmangham** [google/model-viewer#2137] [Q&A]
Hello,

Apologies if this has been answered many times already.

I am finding it unclear on the website and haven't found a straight-forward yes or no in the Q&As section.

The [Browser Support](https://modelviewer.dev/#section-browser-support) section of the website suggests that <model-viewer> is supported on the last 2 major versions of all evergreen desktop and mobile browsers.

I am aware that Safari is not an evergreen browser, but see many Issues reports which suggest it does work.

Please can you confirm if it does indeed work on Safari when **not using ar-modes**?

![image](https://user-images.githubusercontent.com/12933804/111620834-239ed280-87df-11eb-9dcc-fde7a103cd67.png)

Thank you in advance,
Elliott

**Comment by elalish** [comment]
Indeed; I forgot evergreen does not include Safari; I'll update the text. Yes, Safari is supported on both desktop and iOS.

**Reply by elliottmangham** [reply]
Wonderful news! Thank you @elalish

**Comment by haroldao** [comment]
Yes 😋... I've just tested it on my iPhone (with Safari)
